pkg/controller/storagecluster/cephcluster_test.go

```diff
@@ -167,9 +167,10 @@ func TestStorageClassDeviceSetCreation(t *testing.T) {
 	sc1.Spec.StorageDeviceSets = mockDeviceSets
 	sc1.Status.NodeTopologies = &api.NodeTopologyMap{
 		Labels: map[string]api.TopologyLabelValues{
-			zoneTopologyLabel: []string{
-				"zone1",
-				"zone2",
+			defaults.RackTopologyKey: []string{
```

Comment:
Don't modify the existing test cases, only add more test cases. If any of these changes are required for the previous patch to pass unit tests, those changes should be made in the previous commit.

Reply:
Small tweak was required for the existing tests - have added in seperate commit